MSAL.xcworkspace/contents.xcworkspacedata

```diff
@@ -4,7 +4,4 @@
    <FileRef
       location = "group:MSAL/MSAL.xcodeproj">
    </FileRef>
-   <FileRef
-      location = "group:MSAL/MSAL iOS Native Auth E2E Tests.xctestplan">
```

Comment:
Do you know why this shows as removed? I see that the `MSAL iOS Native Auth E2E Tests.xctestplan` is still in the project, so it all should be good, but I'm not sure why this change appears here.